pkgs/development/python-modules/canals/default.nix

```diff
@@ -32,6 +33,7 @@ buildPythonPackage rec {
 
   propagatedBuildInputs = [
     networkx
+    typing-extensions
   ];
 
   passthru.optional-dependencies = {
```

Comment:
There are changes in dependencies.
https://github.com/deepset-ai/canals/commit/2971b4de2aeea28704ad03931e16dd6f642ada8b

Reply:
Opt. dependencies removed. Thanks.